packages/web/README.md

```diff
@@ -44,4 +49,38 @@ This code will make the transaction, let the user sign it using algosigner and s
 
 You can also use `web.sendTransaction()` or `web.signTransaction()` in a react app.
 
-**Note:** We don't support checkpoints yet. Currently `deployASA`, `deploySSC` functions don't work. User should directly pass assetIndex, appIndex instead of asaName, appName.
+### deployApp
+`deployer.deployApp` deploys stateful smart contract.
+### Parameters:
+- approvalProgram:  approval program filename (must be present in assets folder)
+- clearProgram:  clear program filename (must be present in assets folder)
+- flags:  AppDeploymentFlags
+- payFlags:  Transaction Parameters
+- scTmplParams:  Smart contract template parameters (used only when compiling PyTEAL to TEAL). This is an optional parameter.
+- appName: name of the app to deploy. This name (if passed) will be used as
+the checkpoint "key", and app information will be stored agaisnt this name. This is an optional parameter.
```

Comment:
Are you sure about "This is an optional parameter"? I think we changed the design to make it required by default. Could you please check? cc: @robert-zaremba

Reply:
Correct. Both `scTmplParams` and `appName` are optionsl.

                scTmplParams?: SCParams,
                appName?: string

Instead of listing all arguments, I think it's easier and better to link to the API documentation: https://algobuilder.dev/api/algob/interfaces/types.Deployer.html#deployApp

This way it will be always up to date with the released version.

Reply:
Ok, the parameters link has been added.